src/parsers/manifest/dash/js-parser/parse_from_document.ts

```diff
@@ -88,11 +92,14 @@ export default function parseFromDocument(
           ) : IDashParserResponse<string> {
             const resourceInfos : ILoadedXlinkData[] = [];
             for (let i = 0; i < loadedXlinks.length; i++) {
-              const { responseData: xlinkData,
+              const { responseData: xlinkResp,
                       receivedTime,
                       sendingTime,
                       url } = loadedXlinks[i];
-              const wrappedData = "<root>" + xlinkData + "</root>";
+              if (!xlinkResp.success) {
+                throw new Error("DASH parser: Invalid external ressources");
```

Comment:
What was the previous error for that case?
Would be nice to have the same one.
By default, I would guess that it would be `xlinkResp.error`